eng/test-job.yml

```diff
@@ -92,9 +92,7 @@ jobs:
         - name: testhostArg
           value: 'buildtesthostonly'
 
-    # FreeBSD test jobs are disabled since we don't have any FreeBSD helix queues.
-    ${{ if eq(parameters.osGroup, 'FreeBSD') }}:
-      condition: false
+    condition: ${{ parameters.condition }}
```

Comment:
Does explicitly setting it like this override the default of only running if previous job succeeded? If a previous job failed, but `parameters.condition` is true (`_BypassTesting` is not true), will this job run?

Reply:
No. that is the continue on error logic which is different. Is that what you are talking about?

Reply:
Not sure? I think it is related, but not the same. From my read of the docs, `continueOnError` indicates if future jobs should run if this job fails and `condition` indicates if this job should run (where the default is only if the jobs this one depends on succeeded). The condition can be set `always()`, which suggests to me that if the condition evaluates to true, the job would run regardless of the value of `continueOnError` in previous jobs?

Reply:
We might need to do something here like `and(succeeded(), ${{ parameters.condition }})` to make sure we only run the job when there are no failed jobs.